jwst/assign_wcs/niriss.py

```diff
@@ -108,12 +110,20 @@ def niriss_soss(input_model, reference_files):
 
     Notes
     -----
-    It includes tWO coordinate frames -
+    It includes two coordinate frames -
     "detector" and "world".
 
-    It uses the "specwcs" reference file.
+    It uses the "specwcs" reference file type.
     """
 
+    # The input is the grism image
```

Comment:
Which kind of NIRISS "grism" image are we talking about here? The GR700XD grism that's used for SOSS mode, or one of the GR150R or GR150C grisms that's used for WFSS mode?